### ISSUE TITLE
crash when call QApplication::exit()

### ISSUE THREAD

**Issue by yuriyoung** [antonypro/QGoodWindow#2]
crash when call QApplication::exit() to exit application in windows 10.
exception triggered in `QGoodWindow::WndProc`:

example code in mainwindow.cpp:
```
MainWindow::MainWindow(QWidget *parent) : QGoodWindow(parent)
{
    QWidget *centralWidget = new QWidget();
    QHBoxLayout *layout = new QHBoxLayout(centralWidget);
    QPushButton *button = new QPushButton("Button");
    layout->addWidget(button);
    setCentralWidget(centralWidget);
    connect(button, &QPushButton::clicked, this, [](){
        QApplication::quit(); // or exit();
    });
}
```

**Comment by antonypro** [comment]
@yuriyoung: I confirmed this is a bug and i'm working to solve that. I'll update the repository when i have a solution. Thank you for reporting!

**Comment by antonypro** [comment]
@yuriyoung: I fixed this bug on Version 1.0: https://github.com/antonypro/QGoodWindow/blob/9abc0538094a43fcea165a15b70d363f1bc1af25/QGoodWindow/QGoodWindow/QGoodWindowSource/qgoodwindow.cpp#L151

**Comment by antonypro** [comment]
I will consider this issue as solved.
